SQL/0000-00-01-Permission.sql

```diff
@@ -111,7 +111,20 @@ INSERT INTO `permissions` VALUES
     (49,'data_release_upload', 'Data Release: Upload file', 2),
     (50,'data_release_edit_file_access', 'Data Release: Grant other users view-file permissions', 2),
     (51,'instrument_manager_read', 'Instrument Manager: View module', 2),
-    (52,'instrument_manager_write', 'Instrument Manager: Install new instruments via file upload', 2);
+    (52,'instrument_manager_write', 'Instrument Manager: Install new instruments via file upload', 2),
+    (53, 'biobank_specimen_view', 'Biobank: View Specimen Data', 2),                
+    (54, 'biobank_specimen_create', 'Biobank: Create Specimen Data', 2),            
+    (55, 'biobank_specimen_update', 'Biobank: Update Specimen Data', 2),            
+    (56, 'biobank_specimen_alter', 'Biobank: Alter Specimen Data', 2),              
```

Comment:
Is this different from update?

Reply:
yeah - `Update` is the ability to change fields that are regularly updating throughout the specimen lifecycle (e.g. quantity, freeze/thaw cycle, etc.). `Alter` is  the ability to change fundamental aspects of the specimen (e.g. session, type) that should only be changed if there was a significant error in data entry.